native/src/routes/feedback/containers/FeedbackModalContainer.js

```diff
@@ -1,33 +1,296 @@
 // @flow
 
 import * as React from 'react'
-import type { NavigationStackProp } from 'react-navigation-stack'
+import type { NavigationLeafRoute, NavigationScreenProp } from 'react-navigation'
+import { connect } from 'react-redux'
+import { type Dispatch } from 'redux'
 import { withTranslation } from 'react-i18next'
 import withTheme from '../../../modules/theme/hocs/withTheme'
 import FeedbackModal from '../components/FeedbackModal'
 import FeedbackVariant from '../FeedbackVariant'
-import { createFeedbackEndpoint } from 'api-client'
+import {
+  CATEGORIES_FEEDBACK_TYPE, CityModel,
+  CONTENT_FEEDBACK_CATEGORY,
+  createFeedbackEndpoint, DEFAULT_FEEDBACK_LANGUAGE, EVENTS_FEEDBACK_TYPE, INTEGREAT_INSTANCE,
+  OFFER_FEEDBACK_TYPE, OfferModel, OFFERS_FEEDBACK_TYPE,
+  PAGE_FEEDBACK_TYPE,
+  SEARCH_FEEDBACK_TYPE, TECHNICAL_FEEDBACK_CATEGORY
+} from 'api-client'
 import type { FeedbackParamsType } from 'api-client'
 import determineApiUrl from '../../../modules/endpoint/determineApiUrl'
+import type { TFunction } from 'react-i18next'
+import type { StatusPropsType } from '../../../modules/endpoint/hocs/withPayloadProvider'
+import withPayloadProvider from '../../../modules/endpoint/hocs/withPayloadProvider'
+import type { StateType } from '../../../modules/app/StateType'
+import createNavigateToFeedbackModal from '../../../modules/app/createNavigateToFeedbackModal'
+import type { StoreActionType } from '../../../modules/app/StoreActionType'
 
-const TranslatedFeedbackModal = withTranslation('feedback')(withTheme(FeedbackModal))
+type FeedbackType = 'Category' | 'Event' | 'Pois' | 'Offers' | 'Search' | 'Disclaimer'
+
+export type FeedbackInformationType = {
+  type: FeedbackType,
+  isPositiveFeedback: boolean,
+  language: string,
+  cityCode?: string,
+  path?: string,
+  title?: string,
+  feedbackAlias?: string,
```

Comment:
Is this for offers? Is it correctly named `feedbackAlias`?

Reply:
Yes it is used for offers. Renamed to alias.